### PR TITLE
Ret 4284 - Feature flag case file view

### PR DESCRIPTION
**Pull request by marcjudge**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RET-4284

### Change description ###
Feature flag case file view

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField-CaseFileView-nonprod.json.json
Comment (ss-evoco) [inline definitions/json/AuthorisationCaseField/AuthorisationCaseField-CaseFileView-nonprod.json.json:1]: File is called ...-nonprod.json.json

File: definitions/json/Categories/Categories-CaseFileView-nonprod.json
Comment (ss-evoco) [inline definitions/json/Categories/Categories-CaseFileView-nonprod.json:1]: I think you are referring to these changes in CaseField and ComplexTypes. If so, you need to put them in a prod file too.

**Review by ss-evoco — CHANGES_REQUESTED** [review]

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField-CaseFileView-nonprod.json.json
Comment (marcjudge) [reply definitions/json/AuthorisationCaseField/AuthorisationCaseField-CaseFileView-nonprod.json.json:1]: Fixed

File: definitions/json/Categories/Categories-CaseFileView-nonprod.json
Comment (marcjudge) [reply definitions/json/Categories/Categories-CaseFileView-nonprod.json:1]: Done

**Review by ss-evoco — APPROVED** [review]

**Review by JackReevies — APPROVED** [review]

**Pull request merged**